test/inductor/test_compiled_autograd.py

```diff
@@ -1534,6 +1534,7 @@ def fn():
         self.assertEqual(
             sum(1 for e in expected_logs if e in logs.getvalue()), len(expected_logs)
         )
+        torch._logging.set_logs(compiled_autograd_verbose=False)
```

Comment:
These should also be in try-finally blocks so we restore the state even if the test fails.